### DISCUSSION TITLE
Github Commit Error After First Site Deploy

### DISCUSSION THREAD

**Discussion by CamNagle24** [community/community#137650] [New to GitHub]
Hello, I have just deployed my site for the first time and I’m getting this weird error when I try to push my local commits. “The remote disconnected. Check your Internet connection and try again”. I can push commits from other repos so I know it’s not my github but it is the first commit since I deployed the website through netlify. Let me know if you have any advice thanks. It could be that there is a permission error but I have tried re-connecting my repository and that did not fix it.

I have put my repository, my site, and the error below.

https://github.com/CamNagle24/nagle-portfolio
https://cameron-nagle.dev/
![Screenshot 2024-08-30 at 4 00 00 PM](https://github.com/user-attachments/assets/5eaf87e9-f24f-4839-a813-e39c58b224b6)

**Comment by Sin-cy** [comment]
Hey. I use netlify to deploy my websites as well. I never used github desktop though. From the image I see you are currently using Gihub Desktop. I've seen issues like this happening when using Github desktop. I am not sure why. 

The one thing that might actually work is if you try go to your command line and push the changes from there, see if it works

But if you don't want to use command line you could try
- clearing your credential this way if you haven't already `git credential-osxkeychain erase`
then try pushing again and it should prompt you to enter credentials again

- checking if your remote url matches with your Github RepoURL. 
`git remote -v`

- check if there are any LARGE files being added recently because github will reject those pushes

- netlify settings check the build and deploy section see if permissions are all setup correctly. Check if the deploy branch is correctly selected.
